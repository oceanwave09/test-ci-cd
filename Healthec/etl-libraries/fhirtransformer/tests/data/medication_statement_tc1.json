{
  "source_id": "12345689",
  "source_system": "http://www.bmc.nl/portal/prescriptions",
  "status": "active",
  "medication_code": "27658006",
  "medication_display": "Amoxicillin (product)",
  "medication_system": "http://snomed.info/sct",
  "patient_id": "69f946cb-a475-429d-ba9f-9ce0751a1904",
  "effective_period_start": "2015-01-17T16:00:00+10:00",
  "effective_period_end": "2015-01-17T16:30:00+10:00",
  "asserted_date_time": "2015-01-17",
  "reason_code": "65363002",
  "reason_display": "Otitis Media",
  "reason_system": "http://snomed.info/sct",
  "note": "Patient indicates they miss the occasional dose"
}