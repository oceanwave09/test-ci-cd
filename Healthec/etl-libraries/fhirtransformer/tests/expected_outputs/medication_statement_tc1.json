{
  "resourceType": "MedicationStatement",
  "identifier": [
    {
      "type": {
        "coding": [
          {
            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
            "code": "RI",
            "display": "Resource Identifier"
          }
        ]
      },
      "system": "http://www.bmc.nl/portal/prescriptions",
      "value": "12345689"
    }
  ],
  "status": "active",
  "medicationCodeableConcept": {
    "coding": [
      {
        "system": "http://snomed.info/sct",
        "code": "27658006",
        "display": "Amoxicillin (product)"
      }
    ]
  },
  "subject": {
    "reference": "Patient/69f946cb-a475-429d-ba9f-9ce0751a1904"
  },
  "effectivePeriod": {
    "start": "2015-01-17T16:00:00+10:00",
    "end": "2015-01-17T16:30:00+10:00"
  },
  "dateAsserted": "2015-01-17",
  "reasonCode": [
    {
      "coding": [
        {
          "system": "http://snomed.info/sct",
          "code": "65363002",
          "display": "Otitis Media"
        }
      ]
    }
  ],
  "note": [
    {
      "text": "Patient indicates they miss the occasional dose"
    }
  ]
}